src/components/Charts.tsx

```diff
@@ -109,13 +113,16 @@ function getBarChartData(
  * example 'getPatient'
  * @param sites a string list containing 1-n sites that will be shown in the candlechart.
  * example ['s1','s2','s3','s4']
+ * @param siteProps map ecah siteKey to a color
  * @param fileHandler is fuleHandler :)
  * @returns a data structure in correct format to paint a candleChart.
  */
 function getCandleChartData(
   metric: string,
   sites: Array<string>,
-  fileHandler: any
+  fileHandler: any,
```

Comment:
The problem with the tests was that we sent in our siteprops in the parameter of fileHandler. This was hard to spot since filehandler has type any. In the future I dont think we should use any type since we spent many hours debugging this. @NilsArnlund @Dethcrvsh

Reply:
Okay good that you managed to fix the issue. But I dont see how knowing the type of filehandler would have made this debugging easier?

Reply:
Good catch! I opened an issue for creating type definitions for our WebAssembly modules. https://github.com/PUM04/pum-04/issues/76